projects/ion/src/lib/utils/debounce.spec.ts

```diff
@@ -0,0 +1,70 @@
+import debounce from './debounce';
+
+const func = jest.fn();
+jest.useFakeTimers();
+
+describe('debounce', () => {
+  describe('without arguments', () => {
+    beforeEach(() => {
+      func.mockClear();
+    });
+    it('should call function after a given time', () => {
+      debounce(func, 1000)();
+      expect(func).not.toHaveBeenCalled();
+      jest.runAllTimers();
```

Comment:
I did like that

Reply:
![adventure-time-finn](https://user-images.githubusercontent.com/57760325/206543270-bde9fadd-4ea9-4ff6-8190-b0d9b8937926.gif)